docs/src/config/domains.md

```diff
@@ -100,9 +100,9 @@ and orthogonal.
 
 with
 
-`"Index" [None]` :  Index of this lossy domain, used in postprocessing output files.
+`"Index" [None]` :  Index of this bulk domain, used in postprocessing output files.
 
-`"Attributes" [None]` :  Integer array of mesh domain attributes for this lossy domain.
+`"Attributes" [None]` :  Integer array of mesh domain attributes for this bulk domain.
```

Comment:
Does it need `bulk` here? Is this differentiating from a non-bulk (I guess a surface?) domain?